swarm/storage/netstore.go

```diff
@@ -69,22 +68,14 @@ func (self *NetStore) Get(key Key) (chunk *Chunk, err error) {
 	select {
 	case <-t.C:
 		// mark chunk request as failed so that we can retry
-		chunk.SetErrored(true)
+		chunk.SetErrored(ChunkErrNotFound)
 		return nil, ErrChunkNotFound
 	case <-chunk.ReqC:
 	}
-	chunk.SetErrored(false)
+	chunk.SetErrored(ChunkErrOk)
```

Comment:
why not nil again?

Reply:
I don't understand. `ChunkError` is `byte` alias

Reply:
i think errrors should be errors not enums, just define error constants